.github/workflows/deploy.yml

```diff
@@ -43,7 +43,7 @@ jobs:
       - name: Install Julia
         uses: julia-actions/setup-julia@v1
         with:
-          version: 1.6
+          version: 1.10
```

Comment:
```suggestion
          version: '1.10'
```
for yaml `1.10` is the floating point number `1.1` 🙂

Reply:
Nooooo!

Reply:
so 1.11 would work just fine without quotes?

Reply:
Maybe, but I wouldn't trust yaml parsing capabilities to do something sensible 😉

Reply:
"Quotes are only required if you specify a patch version or if the minor version is a multiple of 10"